pkg/transformers/resolver.go

```diff
@@ -0,0 +1,595 @@
+// SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
```

Comment:
In my local tests, I observe that the place of the first resolver function's code comment changes. This is not a very important issue. However, I thought this place could be reviewed in case it is re-visited.

Reply:
Thanks for reporting this unwanted behavior. I've tackled with this issue a bit and found [this](https://github.com/golang/go/issues/20744) upstream issue regarding the floating comments when we modify the existing code. Let me open a separate issue to address this as I suspect we would need to properly calculate the position of that single comment in the transformer source code or use a higher level library like [this one](https://github.com/dave/dst) to fix this and currently, to the best of my knowledge, it does cause any functional issues.